docs/api/session.md

```diff
@@ -831,6 +831,21 @@ this session just before normal `preload` scripts run.
 Returns `String[]` an array of paths to preload scripts that have been
 registered.
 
+#### `ses.setCodeCachePath(path)`
+
+* `path` String - The Code cache location.
+
+Sets code cache directory. By default, the directory will be `Code Cache` under the
+respective user data folder.
+
+#### `ses.clearCodeCaches()`
+
+* `options` Object
+  * `urls` String[] (optional) - An array of url corresponding to the resource that needs to
+    be removed. If the list is empty then all entries in the cache will be removed.
```

Comment:
It looks like the underlying API also supports time limits, I think for e.g. erasing caches older than a certain date. Should we expose that functionality here too?

Reply:
Can certainly provide that, this was based on the other cache clearing apis that also does not expose the time limits.

Reply:
i'm OK with leaving it as is if the extra functionality isn't needed, just thought i'd ask :)